aplib.net-demo/Assets/Scripts/Camera/CinemachinePOVExtension.cs

```diff
@@ -0,0 +1,51 @@
+using UnityEngine;
+using Cinemachine;
+
+public class CinemachinePOVExtension : CinemachineExtension
```

Comment:
```suggestion
public class CinemachinePovExtension : CinemachineExtension
```
3+ letter abbreviations are handles as words